app/routes/userInterface.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+const router = express.Router();
+const jobsModel = require('../models/jobsModel');
+
+/* GET home page. */
+router.get('/', async function(req, res, next) {
+    const data = await jobsModel.getAll();
```

Comment:
You call jobsModel.getAll(), but I don't see the getAll function in jobsModel, so maybe you forgot to add it in.